src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -109,6 +109,32 @@ class NotebookCard extends Component {
       }
     }, printName(name))
 
+    const tenMinutesAgo = _.tap(d => d.setMinutes(d.getMinutes() - 10), new Date())
+    const isRecent = new Date(updated) > tenMinutesAgo
+    const recentEditedInfo = h(PopupTrigger, {
```

Comment:
nit: Since there's nothing interactive in the content, this could be a TooltipTrigger instead.